src/features/unified-share-modal/__tests__/ContactsField-test.js

```diff
@@ -111,6 +152,122 @@ describe('features/unified-share-modal/ContactsField', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'alright',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should match suggested collaborators full names no spaces', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'personalright',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should match suggested collaborators full name', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'person alright',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should match suggested collaborators by email alias', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'bbb',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should not match suggested collaborators with email domain', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'box',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should not match suggested collaborators when search is less than 3 characters', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'ba',
+            });
+            const result = wrapper.instance().addSuggestedContacts([]);
+            expect(result).toMatchSnapshot();
+        });
+
+        test('should match suggested collaborators but not duplicate', () => {
+            const wrapper = getWrapper({
+                suggestedCollaborators: largeSuggestions,
+            });
+            wrapper.setState({
+                pillSelectorInputValue: 'person',
+            });
+            const result = wrapper.instance().addSuggestedContacts(Object.values(largeSuggestions));
+            expect(result).toMatchSnapshot();
```

Comment:
For this test and others, you should add an expectation to ensure the size or some other basic expectation is met. I think this redundancy is nice to have because it's so easy to update snapshot.
`expect(result).toHaveLength(x)`
or you could check for emails
`expect(result.map(contact => contact.email)). arrayContaining(<some-email>)`
or the inverse
`expect(result.map(contact => contact.email)).not. arrayContaining(<some-email>)`